site/en/user_guide.md

```diff
@@ -481,7 +484,8 @@ the existing processors, their tasks and features, see the [next section](#get-m
 
 To get all available processors you might use the autocomplete in your preferred console.
 
-**Note:** Activate virtual environment first.
+**Note:** If you installed OCR-D via Docker make sure you run the bash in the ocrd Docker image as described in the section Preparations[docker-installation].
```

Comment:
```suggestion
**Note:** If you installed OCR-D via Docker make sure you run the bash shell in the ocrd docker image as described in the section [Preparations](#docker-installation).
```

Not sure where you want to link here but the syntax is wrong.

Reply:
I believe what was meant was the "login to Docker" recipe. Only that will get you auto-completion of `ocrd-` in the Docker case.